testdata/helm/memcached-operator/config/manager/manager.yaml

```diff
@@ -28,6 +28,18 @@ spec:
         - "--enable-leader-election"
         - "--leader-election-id=memcached-operator"
         name: manager
+        livenessProbe:
+          httpGet:
+            path: /readyz
+            port: 8081
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 5
+          periodSeconds: 10
```

Comment:
livenessProbe using /readyz endpoint and readinessProbe using /healthz endpoint. Should be the other way around I guess.